canvas_modules/harness/cypress/integration/canvas/context-menu.js

```diff
@@ -88,7 +88,7 @@ describe("Test of context menu", function() {
 		cy.verifyNumberOfSelectedObjects(0);
 	});
 
-	it("Test context menu position is moved correctly with bottom panl open", function() {
+	it.skip("Test context menu position is moved correctly with bottom panl open", function() {
```

Comment:
Please add a comment here to indicate why we are skipping this test. Say something like: 
"This test is being skipped because the canvas area needs to be fixed. At the moment it stretches underneath the bottom panel to the bottom of the common canvas div. It needs to be fixed so that, when the bottom panel is open, the canvas area only extends down to where the bottom panel begins."

Reply:
You seem to have resolved this review comment without adding the comment I suggested to the code.